src/python/pants/backend/python/util_rules/pex_environment.py

```diff
@@ -67,6 +85,25 @@ class PexRuntimeEnvironment(Subsystem):
         advanced=True,
     )
 
+    @property
+    def run_packaged_firstparty_code(self) -> bool:
+        if self.options.is_default("run_packaged_firstparty_code"):
+            warn_or_error(
+                "2.14.0dev0",
+                "the option --pex-run-packaged-firstparty-code defaulting to false",
+                softwrap(
+                    """
+                    In Pants 2.14, running a `pex_binary` will actually package the PEX and run it,
```

Comment:
(The line will be too long w/ this suggestion)

```suggestion
                    In Pants 2.14, by default, running a `pex_binary` will actually package the PEX and run it,
```